.github/workflows/wasmcloud_host.yml

```diff
@@ -268,3 +268,29 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           prerelease: false
           draft: false
+  
+  release-charts:
+    if: ${{ startswith(github.ref, 'refs/tags/') || github.event_name == 'workflow_dispatch' }}
```

Comment:
to think about for future: Do these charts apply only for our current set of releases which include wasmcloud_host? If we started releasing host_core separately, would we need new charts? If yes, we'll need to change this condition to run the workflow only for wasmcloud_host release tags.

Reply:
Yep, that is correct